action_server/src/robocorp/action_server/_server_expose.py

```diff
@@ -27,6 +28,42 @@ class BodyPayload(BaseModel):
     headers: dict
 
 
+class ExposeSessionJson(BaseModel):
+    expose_session: str
+    url: str
+
+
+def get_expose_session_path() -> str:
+    return os.path.join(os.getcwd(), ".robocorp", "expose_session.json")
+
+
+def read_expose_session_json() -> None | ExposeSessionJson:
+    session_json = None
+    try:
+        expose_session_path = get_expose_session_path()
+        log.debug(f"🗂️ Reading expose_session.json path={expose_session_path}")
+        with open(expose_session_path, "r") as f:
+            session_json = ExposeSessionJson(**json.load(f))
+    except FileNotFoundError:
```

Comment:
I think it should also handle json loading errors and issues creating the ExposeSessionJson constructor (i.e.: in case the format changes in the future we don't want errors here in case the user for some reason uses 2 different versions in the same directory).

Reply:
This is actually a really good point. 
Having some way to handle possible changes in the JSON as we can expect things to develop and change at a rapid rate.

Reply:
Added `ValidationError` handler that logs the error.
We're using pydantic model for validating the input, so should be easy to handle changes. We can just set proper defaults for updated fields or implement a new schema for new data shape.

Reply:
@ktaube I think it should also handle JSONDecodeError (i.e.: if by any reason the file becomes corrupted I believe it can be silently ignored instead of showing an exception to the user).